From 320831d52184be682040bcac19daf029b23b9425 Mon Sep 17 00:00:00 2001
From: Paul Barker <pbarker@konsulko.com>
Date: Tue, 12 May 2020 18:12:30 +0100
Subject: [PATCH] arago.conf: Allow TMPDIR path modification to be disabled
To: meta-arago@arago-project.org

We define and use TCMODEAPPEND in the same way TCLIBCAPPEND is defined
and used in oe-core. This allows the modification of the TMPDIR path to
be disabled where needed.

Signed-off-by: Paul Barker <pbarker@konsulko.com>
---
 meta-arago-distro/conf/distro/arago.conf | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/meta-arago-distro/conf/distro/arago.conf b/meta-arago-distro/conf/distro/arago.conf
index 84251d76..6ea41989 100644
--- a/meta-arago-distro/conf/distro/arago.conf
+++ b/meta-arago-distro/conf/distro/arago.conf
@@ -72,7 +72,8 @@ INITRAMFS_FSTYPES_k2l-hs-evm = "cpio.gz"
 # Mask any broken recipes (currently none)
 #BBMASK = ""
 
-TMPDIR .= "-${TCMODE}"
+TCMODEAPPEND ?= "-${TCMODE}"
+TMPDIR .= "${TCMODEAPPEND}"
 
 LIBC_DEPENDENCIES_remove = "glibc-gconv-cp1252 glibc-gconv-ibm850 glibc-gconv-iso8859-1 glibc-gconv-iso8859-15 glibc-localedata-i18n"
 
-- 
2.26.2

